NARFO_BE/NARFO_BE/Models/Club.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace NARFO_BE.Models
+{
+    public partial class Club
+    {
+        public Club()
+        {
+            Member = new HashSet<Member>();
+        }
+
+        public int ClubId { get; set; }
```

Comment:
the table is already called club. There is no reason to start your properties with "Club" this is repetitive and also not following a standard